.env.example

```diff
@@ -9,3 +9,7 @@ DATABASE_NAME=database
 ELASTIC_SEARCH_HOST=http://elasticsearch:9200
 ELASTIC_LOGSTASH_HOST=http://logstash:5000
 ELASTIC_APM_HOST=http://apm:8200
+
+# Redis Configuration
+REDIS_HOST=localhost
```

Comment:
`REDIS_HOST` should use the docker service name. Use `redis` instead of `localhost`